### DISCUSSION TITLE
Make an RTSP server with Mongoose

### DISCUSSION THREAD

**Discussion by drolex2** [cesanta/mongoose#2202] [Ask questions, get answers] [answered]
Is it possible to make an RTSP server with Mongoose? I would use FFmpeg with a live stream output like:
`-f rtsp -rtsp_transport tcp rtsp://localhost:554/v1`
I would have multiple live streams, and I want Mongoose to serve those streams to multiple clients. If this can be done, are there any examples? Can you help me set this up?

**Comment by scaprile** [comment, marked as answer]
Besides the known and advertised application protocols, Mongoose handles TCP and UDP, anything above that is your business.
https://mongoose.ws/documentation/tutorials/tcp/
https://mongoose.ws/documentation/#udp
